extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -126,18 +123,30 @@
   height: 1em !important;
 }
 
-.i-amphtml-story-interactive-slider-input::before {
+.i-amphtml-story-interactive-slider-input-size::before,
+.i-amphtml-story-interactive-slider-input-size::after {
+  content: '' !important;
   background: linear-gradient(
     var(--gradient-angle),
-    white,
-    var(--interactive-accent-color) var(--percentage-corrected)
+    var(--interactive-accent-color) var(--percentage-corrected),
+    transparent var(--percentage-corrected)
   ) !important;
-  content: '' !important;
-  opacity: 0.3 !important;
   width: 100% !important;
   height: 100% !important;
   border-radius: 0.5em !important;
   position: absolute !important;
+  margin: 0em calc(var(--thumb-size) * -0.5) !important;
+  width: calc(100% + var(--thumb-size)) !important;
+}
+
+.i-amphtml-story-interactive-slider-input-size::after {
+  background: linear-gradient(
+    var(--gradient-angle),
+    white,
+    var(--interactive-accent-color) var(--percentage-corrected)
+  ) !important;
+  opacity: 0.3 !important;
+  z-index: 0 !important;
 }
 
 .i-amphtml-story-interactive-slider-input::-webkit-slider-thumb {
```

Comment:
Could the `.i-amphtml-story-interactive-slider-input::-moz-range-thumb` selector be added to this ruleset?

Reply:
There's sth about the selector that breaks when we specify the two rules in the same ruleset like `a, b {}`. I think because Chrome doesn't know about the Firefox prefixes, it invalidates the ruleset. That's why we need 2 separate rulesets here